libs/blocks/tabs/tabs.js

```diff
@@ -146,13 +146,16 @@ const init = (e) => {
   allSections.forEach((e, i) => {
     const sectionMetadata = e.querySelector(':scope > .section-metadata');
     if (!sectionMetadata) return;
-    const metadata = sectionMetadata.querySelectorAll(':scope > div > div');
-    if (metadata[0].textContent === 'tab') {
-      const metaValue = getStringKeyName(metadata[1].textContent);
-      const section = sectionMetadata.closest('.section');
-      const assocTabItem = document.getElementById(`tab-panel-${initCount}-${metaValue}`);
-      if (assocTabItem) assocTabItem.append(section);
-    }
+    const metadatas = sectionMetadata.querySelectorAll(':scope > div');
+    metadatas.forEach((data) => {
```

Comment:
You could rewrite this so that you're not iterating over every piece of metadata - would save a few bytes and is probably faster. 
```
[...metadatas].filter((d) => getStringKeyName(d.children[0].textContent) === 'tab')
  .map((d) => {
    const metaValue = getStringKeyName(d.children[1].textContent);
    const section = sectionMetadata.closest('.section');
    const assocTabItem = document.getElementById(`tab-panel-${initCount}-${metaValue}`);
    if (assocTabItem) assocTabItem.append(section);
  })
```

Reply:
Good call. This is updated